2-mandatory/1-recipes.js

```diff
@@ -22,4 +22,72 @@
   You should write and log at least 5 recipes
 */
 
-// write code here
\ No newline at end of file
+// Recipe 1
+let recipe1 = {
+  title: "Mole",
+  servings: 2,
+  ingredients: ["cinnamon", "cumin", "cocoa"]
+};
+
+console.log(recipe1.title);
+console.log(`Serves: ${recipe1.servings}`);
+console.log("Ingredients:");
+for (let i = 0; i < recipe1.ingredients.length; i++) {
+  console.log(recipe1.ingredients[i]);
+}
+
```

Comment:
.forEch() is also the best mothod.
recipe1.ingredients.forEach(ingredient) => {
console.log(ingredient);
}